examples/deployment/README.md

```diff
@@ -0,0 +1,6 @@
+ASA deployment using Algorand builder from Scale it
```

Comment:
how about removing a scale it and use a link to the repo in `Algorand Builder`?

Reply:
:+1:

Reply:
Changed